helm/cluster-gcp/values.yaml

```diff
@@ -16,8 +16,7 @@ gcp:
     - europe-west3-c
 
 network:
-  autoCreateSubnetworks: true
-  nodeSubnetCidrs: []  # Array of CIDRs to be used if `autoCreateSubnetworks` is set to `false`
+  nodeSubnetCidr: "192.168.0.0/24"  # CIDR to be used for the subnet
```

Comment:
Are we sure this is large enough for all cases?

Reply:
I'll use `/16`